app/src/js/components/home.js

```diff
@@ -147,6 +142,32 @@ class Home extends React.Component {
     }
   }
 
+  pageSection (className, title, children) {
+    return this.section({ className, title, children });
+  }
+
+  sectionHeader (className, title, link) {
+    return this.section({ className, title, link, size: 'large', border: true });
+  }
+
+  section ({ className, title, link, border = false, size = 'medium', children }) {
+    const borderMod = border ? '--border' : '';
+    const sizeMod = `--${size}`;
+    return (
+      <section className={`page__section ${className}`}>
+        <div className='row'>
+          <div className={`heading__wrapper${borderMod}`}>
+            <h2 className={`heading${sizeMod} heading--shared-content--right`}>
+              {title}
+            </h2>
+            {link}
+          </div>
+          {children}
+        </div>
+      </section>
+    );
+  }
```

Comment:
I feel like it might be nice to pull this out of this file and make it a component that can be used across the app

Reply:
I can pull those out, I'm not positive where we can replace them elsewhere in the app.  I see things that look similar, but that can be a next step.

Reply:
2ab4f8bb